packages/nextjs-demo/src/components/Recipe.tsx

```diff
@@ -82,5 +83,12 @@ export function RecipeGenerator({ topic }: { topic: string }) {
     });
   }, [topic]);
 
-  return <div className="whitespace-pre-line">{isLoading ? 'Loading...' : current}</div>;
```

Comment:
lol, this line also killed any sort of incremental streaming, because nothing would be shown until `isLoading` was `false`.